app/frontend/src/cljs/teet/ui/common.cljs

```diff
@@ -592,3 +593,19 @@
         [Paper
          (mapc (r/partial context-menu-item toggle!)
                (remove nil? items))]]]]]))
+
+(defn date-label-component
+  "Shows :label with formatted date from :value with an info icon and :tooltip"
+  [{value :value label :label tooltip :tooltip}]
+  [:div
+   {:style (merge {:color theme-colors/gray} common-styles/body-2-bold)
+    :data-cy "date-label-component"}
+   label
+   [:div
+    {:style common-styles/body-1-bold}
+    [popper-tooltip tooltip
+     [icons/action-info-outlined
+      {:style (common-styles/info-icon-style theme-colors/blue-tab "5px")}]]
```

Comment:
If you want to use this instead of the flex-row this `info-icon-style` should get a map as an arg so the "5px" can be a named value so it would be more clear as to what that means in the calling place.